src/main/java/com/example/NiRi/PasswordJWT.java

```diff
@@ -1,23 +1,21 @@
 package com.example.NiRi;
 
-
 import io.jsonwebtoken.Jwts;
 import io.jsonwebtoken.SignatureAlgorithm;
 import org.springframework.stereotype.Component;
-
 import java.util.Date;
 
 @Component
 public class PasswordJWT {
 
-    private static final String SECRET_KEY = "yourSecretKey"; // Replace with a secure key
+    private static final String SECRET_KEY = "aishwaryaniri221402";
```

Comment:
we should take the security key from application.properties

Reply:
added in application.properties

Reply:
I don't see any changes after these comments. Did you push your updates?